config/rbac/topolvm_node_scc.yaml

```diff
@@ -0,0 +1,30 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: topolvm-node
+  annotations:
+    kubernetes.io/description: 'privileged allows access to all privileged and host
+        features and the ability to run as any user, any group, any fsGroup, and with
+        any SELinux context.  WARNING: this is the most relaxed SCC and should be used
+        only for cluster administration. Grant with caution.'
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: true
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: true
+readOnlyRootFilesystem: false
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+fsGroup:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+volumes:
+  - 'configMap'
+  - 'emptyDir'
+  - 'secret'
+  - 'hostPath'
```

Comment:
How is this SCC associated with the topolvm-node Service account?  There is a `user` filed that contains the list of serviceAccounts. For example- [rook](https://github.com/rook/rook/blob/fffb86295600196389de81905c3402f9aec6e751/deploy/examples/operator-openshift.yaml#L42)

Reply:
- thanks for finding this, I saw `users` field somewhere and forgot where that was during our meetings
- in current scenario we are using ClusterRole & ClusterRoleBinding to use this SCC
- in merged commit it was using `privileged` directly and this changes that to use `topolvm-node` named SCC

Reply:
@sp98: I have include the different settings reading [what they do ](https://docs.openshift.com/container-platform/4.9/rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.html). 
As @leelavg commented we are using a cluster role binding to to associate the sa with the scc. Include the user in the scc seems to link the scc to a single namespace (because the way you need to specify the user), which i think is very limited for our needs.

Reply:
The operator only needs the permissions for the sa in the operator namespace as that is where it will create the containers. Can we try this with a Role?

Reply:
@nbalacha : We cannot use Role instead of Cluster Role, because we need permissions for cluster-scoped resources (like nodes).